content/en/tutorials/securely_accessing_network_locally.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Securely Accessing Opta Networks Locally"
+linkTitle: "Securely Accessing Opta Networks Locally"
+date: 2022-01-03
+draft: false
+description: >
+  Instructions to integrate with the CI/CD platform of your choice
+---
+
+# Overview
+When working with secure private networks, and specifically with the databases in them, one may wish to somehow manually 
+connect to them for debugging purpose, but there is This action, naturally, has security concerns and there's many 
+solutions out there. For our users, we typically recommend installing VPN servers within your opta-managed VPCs and 
+handling network access to your environments from there. For just getting started, however, there's a rather 
+straight-forward TCP proxy solution. This article will discuss both solutions and the security concerns.
+
+# TCP-Proxy Service
+A quick way to gain connectivity is to create a TCP-Proxy service with opta and using `kubectl port-forward` to gain
+access to the proxy locally. Basically, there is this [docker image called tecnativa/tcp-proxy](https://hub.docker.com/r/tecnativa/tcp-proxy)
```

Comment:
I wonder if we should host this image, vs referring to it.
1. not maintained in more than 3 years
2. docker hub could create some issue because of their rate limiting
3. trust opta vs trust tecnativa

An alternative would be to clone the repo and push it the image to github (using gha) to fix 2.

Reply:
@legalosLOTR @NitinAgg ?

Reply:
I like the idea of hosting it on GitHub